SWLOR.Game.Server/ChatCommand/GetFP.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace SWLOR.Game.Server.ChatCommand
 {
-    [CommandDetails("Gets the current FP of player.", CommandPermissionType.Player)]
+    [CommandDetails("Gets the current FP of player.", CommandPermissionType.Player | CommandPermissionType.DM | CommandPermissionType.Admin)]
```

Comment:
DMs/Admins don't have DB entries so just adding them to this won't work. Also they don't have FP.

Reply:
no this is making it so DM's can use it on their player characters

Reply:
In that case you need to add a check to the method to ensure it doesn't run when they're on their DM characters. It's going to error out because there won't be a record in the DB for them.

Reply:
Should be done in the code already ^^